tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -250,23 +201,19 @@ def test_it(self, pyramid_request, parsed_params, is_canvas):
 
 
 class TestHUser:
-    def test_username_is_a_30_char_string(self, pyramid_request):
-        username = LTILaunchResource(pyramid_request).h_user.username
-
-        assert isinstance(username, str)
-        assert len(username) == 30
-
-    def test_userid(self, pyramid_request):
-        userid = LTILaunchResource(pyramid_request).h_user.userid
-
-        assert userid == "acct:16aa3b3e92cdfa53e5996d138a7013@TEST_AUTHORITY"
+    def test_it(self, pyramid_request):
+        assert LTILaunchResource(pyramid_request).h_user == HUser(
+            authority=pyramid_request.registry.settings["h_authority"],
+            username="16aa3b3e92cdfa53e5996d138a7013",
+            display_name=pyramid_request.lti_user.display_name,
+        )
```

Comment:
This is what remains of the tests for `LTILaunchResource.h_user()`: just one tiny test is enough